wikiclue/src/components/dailyEnd.svelte

```diff
@@ -1,44 +1,55 @@
 <script lang="ts">
-    import Overlay from "./overlay.svelte";
-	import { writable } from "svelte/store";
-    export let levelOpen = false;
+	import { goto } from '$app/navigation';
+	import Overlay from './overlay.svelte';
+	import { writable } from 'svelte/store';
+	export let levelOpen = false;
 
-    let statistics = {'Played' : 24, 'Win %': '85%', 'Current Streak': 3, 'Max Streak': 7};
-    let distribution = [4, 1, 3, 3, 4, 1];
-    let max = 4;
-    const isOverlayOpen = writable(false);
+	export let statistics = { Played: 24, 'Win %': '85%', 'Current Streak': 3, 'Max Streak': 7 };
+	export let distribution = [4, 1, 3, 3, 4, 1];
+	let max = 4;
+	const isOverlayOpen = writable(false);
 </script>
 
+<Overlay
+	header="Daily Challenge"
+	onClose={() => {
+		isOverlayOpen.set(false);
+		levelOpen = false;
+		goto('/home');
+	}}
```

Comment:
I don't think we should display the x here

Reply:
So just ignore figma design then?

Reply:
What happens if u press x tho? Does it take u back to the main menu or replay the daily (which would just mean u do it really fast). If u just press x and it goes back to the menu, why do we need 2 buttons that do the same thing?